src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -755,8 +755,24 @@ public function get_test_php_version() {
 			'test'        => 'php_version',
 		);
 
+		if ( ! $response ) {
+			$result['description'] .= sprintf(
```

Comment:
I think the description here should mostly remain unchanged. For most users, telling them that the minimum version could not be determined due to a communication error is not that helpful. The main thing to do here is to return the array early, since there is nothing more that we can determine.

Related to my earlier comment, if we removed the ternary from the original description field, we could update the description after this if statement has completed so that we then add the minimum recommended version of PHP info based on the $response value.

Reply:
Maybe we should test have failed status/try later 
We can say it passed as we don't know